src/interfaces/IRVault.sol

```diff
@@ -49,4 +49,26 @@ interface IRVault is IVault {
     /// @notice Update and Claim all rewards for caller
     function getAllRewards() external;
 
+    /// @notice Update and Claim rewards for specific token
+    function getReward(uint rt) external;
+
+    /// @notice Return reward per token ratio by reward token address
+    ///                rewardPerTokenStoredForToken + (
+    ///                (lastTimeRewardApplicable - lastUpdateTimeForToken)
+    ///                 * rewardRateForToken * 10**18 / totalSupply)
+    function rewardPerToken(uint rewardTokenIndex) external view returns (uint);
```

Comment:
Need add natscpec `/// @param ..` and `/// @return ...` to this and other added by youi methods to this interface